src/Policy/LimitingAll.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\RateLimiter\Policy;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+final class LimitingAll implements LimitingPolicy
```

Comment:
```suggestion
final class LimitAlways implements LimitingPolicy
```

Alternatives are:

- Any.
- Every.

Reply:
`All` здесь означает всех, а не всегда. Как антоним к `лимит на юзера`.

Reply:
What about `Everybody`? It is more descriptive than `All`.